app/src/views/HomeView.vue

```diff
@@ -9,7 +9,7 @@
     </p>
 
     <button class="mt-6 btn btn-secondary">
-      <RouterLink to="/teams">Tip your Team</RouterLink>
+      <RouterLink to="/teams" data-testid="tip-your-team-button">Tip your Team</RouterLink>
```

Comment:
Why you use data-testid ?
isn't data-test enough?

Reply:
Oops i think i dont need to define that, I choose to use data-testid because there is a function in playwright to get by data-test-Id https://playwright.dev/docs/other-locators#id-data-testid-data-test-id-data-test-selectors, but I haven't try if we define it data-test